lib/client-api/src/decorators.ts

```diff
@@ -15,14 +15,25 @@ const defaultContext: StoryContext = {
   globals: {},
 };
 
+/**
+ * When you call the story function inside a decorator, e.g.:
+ *
+ * ```jsx
+ * <div>{storyFn({ foo: 'bar' })}</div>
+ * ```
+ *
+ * This will override the `foo` property on the `innerContext`, which gets
+ * merged in with the default context
+ */
+export const decorateStory = (storyFn: StoryFn, decorator: DecoratorFunction) => {
+  return (context: StoryContext = defaultContext) =>
+    decorator(
+      // You cannot override the parameters key, it is fixed
```

Comment:
Can we make an issue (7.0 I guess) to add some more things to this list? Doesn't make sense to be able to override `storyId` as an example.